src/main/java/webserver/data/Todos.java

```diff
@@ -0,0 +1,14 @@
+package webserver.data;
+
+import java.util.List;
+import java.util.Map;
+
+import webserver.models.Todo;
+
+interface Todos {
```

Comment:
Great!  I suspect having this interface will make testing easier and would also be more flexible if we needed to store TODOs differently.